src/request/containers/RequestDetailPanelDataContainer.ts

```diff
@@ -0,0 +1,21 @@
+import { DataComponent, IDataComponentProps } from '../components/RequestDetailPanelData';
+import { IRequestState } from '../stores/IRequestState';
+import { getTotalOperationCount } from '../selectors/RequestDetailDataSelectors';
+
+import { connect } from 'react-redux';
+
+function mapStateToProps(state: IRequestState): IDataComponentProps {
+    return {
+        totalOperationCount: getTotalOperationCount(state)
+    };
+}
+
+function mapDispatchToProps(dispatch) {
+    return {
+    };
+}
+
+export = connect(
+    mapStateToProps,
+    mapDispatchToProps
+)(DataComponent);
```

Comment:
Given that `mapDispatchToProps` doesn't return anything, I think `connect` can be updated to just be `connect(mapStateToProps)`. But if you have it like this because you are expecting there to be more actions that come along then thats fine.